h2/src/test/scala/org/scalarelational/InheritanceSpec.scala

```diff
@@ -1,60 +0,0 @@
-package org.scalarelational
-
-import org.scalarelational.column.property.{AutoIncrement, PrimaryKey}
-import org.scalarelational.h2.{H2Datastore, H2Memory}
-import org.scalarelational.table.Table
-import org.scalatest.{Matchers, WordSpec}
-
-/**
- * @author Tim Nieradzik <tim@kognit.io>
- */
-class InheritanceSpec extends WordSpec with Matchers {
```

Comment:
Please reintroduce this spec as it checks whether inheritance works with ``to``. You could move it to the ``mapper`` sub-project though.